tests/cephadm/test_collocate_rgw_node_count.py

```diff
@@ -0,0 +1,35 @@
+from ceph.ceph import CommandFailed
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Collocate_RGW_Node_Count(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Deploy collocated RGW with just node count
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
```

Comment:
Add details which variables are expected in `**kw`

Reply:
Thanks @vamahaja
Updated